httpapi/coreapi/accounts_api.go

```diff
@@ -136,3 +136,77 @@ func (h handler) GenerateAccount(w http.ResponseWriter, r *http.Request) {
 	render.Status(r, http.StatusOK)
 	render.JSON(w, r, toClientAccount(acc))
 }
+
+// GetAccounts returns all the accounts in the node.
+// @summary Returns all the accounts in the node.
+// @description Returns all the accounts in the node.
+// @id get_accounts
+// @tags Accounts
+// @produce json
+// @Failure 500 {object} httputils.HTTPError
+// @success 200 {object} coreapi.Accounts
+// @router /v1/accounts [get]
+func (h handler) GetAccounts(w http.ResponseWriter, r *http.Request) {
+	var err error
+	var code int
+	defer httputils.RespondIfError(&code, &err, w, r)
+
+	accs, err := h.srv.GetAccounts()
+	if err != nil {
+		code = http.StatusInternalServerError
+		log.Error(err)
+		return
+	}
+
+	render.Status(r, http.StatusOK)
+	render.JSON(w, r, toClientAccounts(accs))
+}
+
+// CreateAccount creates a new account.
+// @summary Creates a new account.
+// @description Creates a new account.
```

Comment:
could be good to have a bit more documentation about the difference between this method and GenerateAccount above (be more specific about the use case for each method)